invenio_pages/assets/semantic-ui/js/invenio_pages/search/SearchResultItem.js

```diff
@@ -0,0 +1,122 @@
+/*
+ * This file is part of Invenio.
+ * Copyright (C) 2023 CERN.
+ *
+ * Invenio is free software; you can redistribute it and/or modify it
+ * under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+import PropTypes from "prop-types";
+import React, { Component } from "react";
+import { Table } from "semantic-ui-react";
+import isEmpty from "lodash/isEmpty";
+import { Actions } from "@js/invenio_administration";
+import { withState } from "react-searchkit";
+import { AdminUIRoutes } from "@js/invenio_administration";
+import { Formatter } from "@js/invenio_administration";
+import _truncate from "lodash/truncate";
+
+class SearchResultItemComponent extends Component {
+  refreshAfterAction = () => {
+    const { updateQueryState, currentQueryState } = this.props;
+    updateQueryState(currentQueryState);
+  };
+
+  render() {
+    const {
+      title,
+      resourceName,
+      result,
+      columns,
+      displayEdit,
+      displayDelete,
+      actions,
+      idKeyPath,
+      resourceSchema,
+      listUIEndpoint,
+    } = this.props;
+    const resourceHasActions =
+      displayEdit || displayDelete || !isEmpty(actions);
+    result.content = _truncate(result.content, { length: 250 });
```

Comment:
question: the input prop is being mutated by the component. So for a result item with length > 250, does this code truncate the content and never allows the user to read its full value?